.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -242,44 +242,9 @@ jobs:
             --flush-ct
 
       - name: Rotate IPsec Key & Test (${{ join(matrix.*, ', ') }})
-        uses: ./.github/actions/conn-disrupt-test
+        uses: ./.github/actions/ipsec-key-rotate
         with:
-          job-name: conformance-ipsec-e2e-key-rotation-${{ matrix.name }}
-          operation-cmd: |
-            KEYID=$(kubectl get secret -n kube-system cilium-ipsec-keys -o go-template --template={{.data.keys}} | base64 -d | cut -d' ' -f1)
-            if [[ $KEYID -ge 15 ]]; then KEYID=0; fi
-
-            if [[ "${{ matrix.key-two }}" == "gcm(aes)" ]]; then
-              key="rfc4106(gcm(aes)) $(dd if=/dev/urandom count=20 bs=1 2> /dev/null | xxd -p -c 64) 128"
-            elif [[ "${{ matrix.key-two }}" == "cbc(aes)" ]]; then
-              key="hmac(sha256) $(dd if=/dev/urandom count=32 bs=1 2> /dev/null| xxd -p -c 64) cbc(aes) $(dd if=/dev/urandom count=32 bs=1 2> /dev/null| xxd -p -c 64)"
-            else
-              echo "Invalid key type"; exit 1
-            fi
-            data="{\"stringData\":{\"keys\":\"$((($KEYID+1))) ${key}\"}}"
-            kubectl patch secret -n kube-system cilium-ipsec-keys -p="$data" -v=1
-
-            # Wait until key rotation starts
-            while true; do
-              keys_in_use=$(kubectl -n kube-system exec daemonset/cilium -c cilium-agent -- cilium-dbg encrypt status | awk '/Keys in use/ {print $NF}')
-              if [[ $keys_in_use == 2 ]]; then
-                break
-              fi
-              echo "Waiting until key rotation starts (seeing $keys_in_use keys)"
-              sleep 30s
-            done
-
-            # Wait until key rotation completes
-            # By default the key rotation cleanup delay is 5min, let's sleep 4min before actively polling
-            sleep $((4*60))
-            while true; do
-              keys_in_use=$(kubectl -n kube-system exec daemonset/cilium -c cilium-agent -- cilium-dbg encrypt status | awk '/Keys in use/ {print $NF}')
-              if [[ $keys_in_use == 1 ]]; then
-                break
-              fi
-              echo "Waiting until key rotation completes (seeing $keys_in_use keys)"
-              sleep 30s
-            done
```

Comment:
Commit `gh/actions: Add ipsec-key-rotate` is very likely to cause conflicts during backporting. IIUC it only moves the code from the workflow to the action and increases the timeout. Would it be possible to describe that in the commit description so that folks handling conflicts are reassured that they can simply move the code?

Reply:
:+1: Added the note to the commit msg.